crates/rust-analyzer/src/handlers/notification.rs

```diff
@@ -152,13 +154,15 @@ pub(crate) fn handle_did_save_text_document(
         // Re-fetch workspaces if a workspace related file has changed
         if let Some(abs_path) = vfs_path.as_path() {
             if reload::should_refresh_for_change(abs_path, ChangeKind::Modify) {
-                state
-                    .fetch_workspaces_queue
-                    .request_op(format!("workspace vfs file change saved {abs_path}"), false);
+                state.fetch_workspaces_queue.request_op(
+                    format!("workspace vfs file change saved {abs_path}"),
+                    (Some(abs_path.to_owned()), false),
```

Comment:
I was lazy here; I should make this a proper struct instead of a tuple. For context, I'm using this in order to pass the saved file over to `GlobalState::fetch_workspaces` so that I can kick off the `discover_handle` to do its thing.